release/gcb/generate_manifest.sh

```diff
@@ -88,6 +91,14 @@ EOF
         echo "inconsistent shas ISTIO_HEAD_SHA         $ISTIO_HEAD_SHA != $ISTIO_REPO_SHA ISTIO_REPO_SHA" 1>&2
         exit 18
       fi
+      checkout_code "cni" "HEAD" .
```

Comment:
I chatted with @rkpagadala. This consistency check implementation was wrong. This was meant for making sure istio and proxy share the same api dependency, and this should not be checking if api and proxy are at HEAD.

Since the dependency from istio to cni is a direct uni-directional dependency, I don't think we need any consistency check...